### PR TITLE
Migrate from packr to fs

### PR DESCRIPTION
**Pull request by fasmat**
Again another upgrade away from packr to io/fs.

This is a breaking change again, so the version should be bumped to V6

Additionally there is a dependency problem in `gobuffalo/pop` that prevents pop from using newer versions of `gobuffalo/fizz`. This problem was introduced with the replacement directive in the `go.mod` file where now `fizz` depends (transitively via `pop` `v5.3.4`) on a non existing version `v2.0.3+incompatible` of `github.com/mattn/go-sqlite3`. At the same time `pop` breaks if we upgrade to `v2.0.3+incompatible` of `github.com/mattn/go-sqlite3`.

In summary: avoid using replace in `go.mod` files outside of local development. These break your dependency graphs.

For pop to use the latest version of `fizz` we need to update `fizz` to use a newer version of `pop` that doesn't have the replace statement any more.

### CODE REVIEW

**Review by paganotoni — APPROVED** [review]
This looks good to me. Thanks @fasmat !
